pkgs/development/libraries/librist/default.nix

```
@@ -0,0 +1,45 @@
+{ stdenv
+, lib
+, fetchFromGitLab
+, meson
+, ninja
+, cmake
+, pkg-config
+, cjson
+, cmocka
+, mbedtls
+}:
+
+stdenv.mkDerivation rec {
+  pname = "librist";
+  version = "0.2.6";
+
+  src = fetchFromGitLab {
+    domain = "code.videolan.org";
+    owner = "rist";
+    repo = "librist";
+    rev = "v${version}";
+    hash = "sha256-EgWEMRiBJGq7ILmRqdL7Ok6/FhZNmYgiYyWHUhTZqCA=";
+  };
+
+  nativeBuildInputs = [
+    meson
+    ninja
+    pkg-config
+    cmake
+  ];
+
+  buildInputs = [
+    cjson
+    cmocka
+    mbedtls
+  ];
+
+  meta = with lib; {
+    description = "A library that can be used to easily add the RIST protocol to your application.";
+    homepage = "https://code.videolan.org/rist/librist";
+    license = licenses.bsd3;
```

```suggestion
    license = with licenses; [ bsd2 mit isc ];
```
This mirrors what [Debian uses](https://metadata.ftp-master.debian.org/changelogs//main/libr/librist/librist_0.2.7+dfsg-1_copyright) and looks reasonable